cla_frontend/assets-src/stylesheets/design-patterns/_notes.scss

```diff
@@ -3,6 +3,7 @@
 
   textarea {
     padding-right: 30px;
+    min-height: 220px;
```

Comment:
Can we change the rows attr instead of using this? I think it's currently 6, bump it to 8/9